File: edition/2017/cpp/Data_structures/linear_search.cpp
/**
*linear search
*
*
**/


#include <iostream>
#include <stdlib.h>

using namespace std;

int linear_search(int arr[], int ele);
int main()
{
    int arr[] = {2, 3, 4 ,6 ,7};
    int ans = linear_search(arr, 7);
    if(ans == 0)
        cout<<"found!\n";
    else
        cout<<"not found!\n";
    return 0;
}
int linear_search(int arr[], int ele)
{
    int i;
    int len = sizeof(arr);
    for(i = 0; i < len; i++)
    {
        if(arr[i] == ele)
            return 0;
    }
    return 1;
}

File: php/algorithms/linear_search.php
<html>
<head>
<title>linear search</title>
</head>
<body>
<?php 

    function linear_search($arr,$k)
    {
        $n=count($arr);
        for($i=0;$i<$n;$i++)
        {
            if($arr[$i]==$k)
            {
                return $i+1;
                echo "<script>alert('Index of ".$k." is ".$i+1 ."')</script>";
            }
        }
        return -1;
        echo "<script>alert('Element not found!')</script>";
    }

?>
</body>
</html>
